scripts/copy_dummy_assets.sh

```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/env bash
+
+# INSTRUCTIONS:
+# This script copies all dummy-assets to domain/src/main/assets folder and delete
+# the original oppia assets.
+#
+# Run the script from the oppia-android root folder:
+#
+#   bash scripts/copy_dummy_assets.sh
+#
+# NOTE: This script should only be run after running the scripts/copy_oppia_assets.sh
+
+destination_assets_path="domain/src/main/assets"
+
+# Remove current assets from domain/src/main/assets
+rm -r domain/src/main/assets/*
```

Comment:
Suggest using ``$destination_assets_path`` here instead of repeating the path.

Reply:
Updated code